mne/tests/test_cov.py

```diff
@@ -116,14 +116,31 @@ def test_cov_estimation_on_raw_segment():
     picks = pick_channels(raw.ch_names, include=raw.ch_names[:5])
     cov = compute_raw_covariance(raw, picks=picks)
     assert_true(cov_mne.ch_names[:5] == cov.ch_names)
-    assert_true(linalg.norm(cov.data - cov_mne.data[picks][:, picks],
-                ord='fro') / linalg.norm(cov.data, ord='fro') < 1e-4)
+    assert_snr(cov.data, cov_mne.data[picks][:, picks], 100)
     # make sure we get a warning with too short a segment
     raw_2 = raw.crop(0, 1)
     with warnings.catch_warnings(record=True) as w:
         warnings.simplefilter('always')
         cov = compute_raw_covariance(raw_2)
-    assert_true(len(w) == 1)
+    assert_true(any('Too few samples' in str(ww.message) for ww in w))
+
+
+@slow_test
+@requires_sklearn_0_15
+def test_cov_estimation_on_raw_reg():
+    """Test estimation from raw with regularization
```

Comment:
single line docstrings should have the ending quotes in the same line. Sorry, it's a nitpick :)